eZ/Bundle/EzPublishRestBundle/Resources/config/default_settings.yml

```diff
@@ -4,3 +4,83 @@ parameters:
 
     # URI part that all REST routes begin with. By this a REST request is recognized.
     ezpublish_rest.path_prefix: /api/ezp/v2
+
+    ezpublish_rest.root_resources:
```

Comment:
Why don't you write directly to the siteaccess aware container config ? You could then skip loading the container config altogether: https://github.com/ezsystems/ezpublish-kernel/pull/1619/files#diff-0b45d72cefd7a4d4849d6d15e1527b31R40.

```
ezsettings.default.rest_root_resources:
    content:
        ...
```